frontend/public/components/container.jsx

```diff
@@ -158,6 +158,9 @@ const Details = (props) => {
 
   const status = getContainerStatus(pod, container.name) || {};
   const state = getContainerState(status) || {};
+  const stateValue = state.label.toLowerCase() === 'terminated' && _.isFinite(state.exitCode)
```

Comment:
Add a separate field for the original state value rather than calling `toLowerCase()` here. This component shouldn't have any knowledge of how the label is calculated in `getContainerState`.